### ISSUE TITLE
Bug from weekly view when scrolling down to the bottom of the page

### ISSUE THREAD

**Issue by tiffany-tang** [actnforchildren/mental_health_app#115]
This is a curious one as it never happened before. 

But whenever I tried to scroll down the weekly log for 29th Oct - 4th Nov, as soon as I hit the bottom the app will automatically take me to the week before.

@iteles I've attached a video to the email I just sent you as it's easier to show how this bug happened.

**Comment by SimonLab** [comment]
@tiffany-tang I haven't manage to reproduce the error on my phone.
We are catching any swipe event on the card to go on the next/previous week, do you think it's possible that when you scroll, the swipe event was trigger?

**Comment by tiffany-tang** [comment]
@SimonLab I have sent a video of the screen recording to ines, I'm happy to forward that over to you?

**Comment by tiffany-tang** [comment]
@SimonLab Sent you an email because I couldn't attach a .mov here. 

"Hey Simon,

Attached is the video mentioned on Github. You'll see when I start to scroll down the 29 Oct - 4 Nov page, it automatically jumps to the page for the week before. 
This happens for both Safari and Chrome on both iOS and android devices. It also happened this morning when I tested for the same issue. 

Tiffany. "

**Comment by SimonLab** [comment]
Thanks @tiffany-tang we identify the bug. It's due to the fact that a vertical scroll is never perfectly vertical and our `swipe` function manage to catch the horizontal change too (ie scrolling is more like a diagonal movement which trigger the swipe). We try to see if we can calibrate the swipe function

**Comment by tiffany-tang** [comment]
@SimonLab the bug is still happening, both on weekly and monthly view, but not as frequent as previously. It will happen when i first scroll down the page and then will stop.

**Comment by tiffany-tang** [comment]
@SimonLab sorry fyi - I was testing this on both safari and chrome, this is only happening on safari.

**Comment by tiffany-tang** [comment]
@SimonLab This is still happening on Safari.

**Comment by SimonLab** [comment]
@tiffany-tang we just try on @iteles iphone and everything looks fine. I'm wondering if the version you are testing on is not a cached (old) version. Could you try to close Safari completely and reload the application to see if the issue is still here?

**Comment by tiffany-tang** [comment]
@SimonLab 
I have logged out of my current account, closed Safari, and reopened it again and it's still happening. 

My phone is an iphone 8 and I am on version 11.4.1 (15G77)

hope this helps.

**Comment by tiffany-tang** [comment]
@SimonLab Just to clarify, can I go through the different pages by swiping the screen?

**Comment by SimonLab** [comment]
@tiffany-tang yes you should be able to go to the next and previous page (weekly and monthly) with a horizontal swipe

**Comment by SimonLab** [comment]
@tiffany-tang could you test it again? I just deploy a small change
